src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java

```diff
@@ -931,7 +939,44 @@ private JButton getDischargeButton() {
 		});
 		return dischargeButton;
 	}
-
+	
+	/**
+	 * this method creates the button that delete the last charge movement 
+	 * 
+	 * @return
+	 */
+	private JButton getDeleteLastMovementButton() {
+		JButton deleteMovementButton = new JButton(MessageBundle.getMessage("angal.medicalstock.delete.btn"));
+		deleteMovementButton.setMnemonic(KeyEvent.VK_D);
```

Comment:
You should be using:
       angal.common.delete.btn.key                                                                       
       angal.common.description.txt

The way you have it currently it doesn't support translation

Reply:
@ArnaudFonzam after the change suggested by @dbmalkovsky, you should then remove line 949, right?

Reply:
@mwithi No, it think it should be more like:

JButton deleteMovementButton = new JButton(MessageBundle.getMessage("angal.common.delete.btn"));
deleteMovementButton.setMnemonic(MessageBundle.getMessage("angal.common.delete.btn.key");

Reply:
@dbmalkovsky there is a GitHub refresh problem, I can already see part of the the change:

![image](https://github.com/informatici/openhospital-gui/assets/2938553/6475b53b-27d1-4f18-bced-27435cd38923)

Reply:
Why you marked as resolved without applying suggestions ("remove line 949") or without saying why you won't apply it?